plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -771,6 +771,25 @@ int restore_hsakmt_shared_mem(const uint64_t shared_mem_size, const uint32_t sha
 	return 0;
 }
 
+static int pause_process(int fd, const bool enable)
```

Comment:
maybe call it freeze_thaw_queues/pipeline?

Reply:
pause matches the IOCTL name

Reply:
I also don't like the name. Is PAUSE what we agreed on for the ioctl name? "freeze" is a more intuitive name IMO.

Reply:
Yes, pause was the name agreed for the ioctl name. If we change this function, then I think the ioctl name needs to change as well.